README.md

```diff
@@ -1,3 +1,11 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+Create the following environment variables for the code to run:
+
+./.env.test
+
+PGDATABASE = nc_news_test;
+
+./.env.dev
+
+PGDATABASE = nc_news;
```

Comment:
Please add the command to install 'dotenv', also the instructions on how to gitignore those .env file.